### DISCUSSION TITLE
i3 can't connect to X11 on Pop OS 22.04

### DISCUSSION THREAD

**Discussion by davidrutland** [i3/i3#4961] [Q&A] [answered]
## I'm submitting a…

<pre>
[x] Bug
[ ] Feature Request
[ ] Documentation Request
[ ] Other (Please describe in detail)
</pre>

## Current Behavior
Selecting i3 as WM results in black screen with no mouse, prompts, or any response from keyboard input 

## Expected Behavior

Expecting prompt to auto generate config

## Reproduction Instructions

Fresh install of Pop OS 22.04
install i3 metapackage with sudo apt install i3
logout
Select i3 and attempt to log in.

## Environment

Output of `i3 --moreversion 2>&-`:

i3 --moreversion 2>&-

Binary i3 version:  4.20.1 (2021-11-03) © 2009 Michael Stapelberg and contributors

Running version: < 4.2-200

<details><summary>Config file</summary><pre>
</pre>
</details>

Not generated

<pre>

i3-dump-log: ERROR: Cannot connect to X11.

FYI: The DISPLAY environment variable is set to ":0".
Data is not an i3 log file.

</pre>

<pre>
- Linux Distribution & Version:  Pop OS 22.04
- Are you using a compositor (e.g., xcompmgr or compton):
</pre>

https://github.com/pop-os/cosmic-comp

**Comment by Airblader** [comment]
I'm going to convert this to a discussion because it seems like more of a setup issue than a bug in the software (at least I see no evidence thereof). It's probably worthwhile crossposting this to the PopOS community as it depends on what they use to manage sessions. It doesn't seem like i3 is even running based on this report.

**Comment by davidrutland** [comment]
Further information:

i3 works fine with nouveau drivers - using proprietary nvidia drivers results in the above behaviour

**Comment by davidrutland** [comment, marked as answer]
Now solved - yes, it was a config issue

Log into standard pop os cosmic

roll back to nouveau drivers w/ sudo apt install xserver-xorg-video-nouveau

log into i3

auto-generate config etc

reinstall nvidia drivers w/ sudo apt install nvidia-driver-510

nano .config/i3/config

add this:

exec xrandr --setprovideroutputsource modesetting NVIDIA-0
exec xrandr --auto 

reboot

log into i3

**Reply by ghost** [reply]
I just added the xrandr  you have on the bottom with my ubuntu 22 fresh install, and was able to login to i3 afterwards. Thanks for sharing
